phases/phase-1/challenges/refer.md

```diff
@@ -0,0 +1,38 @@
+# Refer a friend
+:::tip Note  
+This challenges allows you to refer one or more friend(s) and get **200 Desmos Tokens per invited user that has completed the [Post challenge](create-post.md) and [accepted your referral](accept-referral.md)**.  
+
+Please note that you can refer multiple friends and get 200 Desmos Tokens per friend referred.
+
+If one of the users who have referred does **not** create a post or accept your referral, you will **not** be granted any Desmos Token. 
+:::
```

Comment:
This challenges allows you to refer one or more friend(s) and get **200 Desmos Tokens per invited user that has completed the [Post challenge](create-post.md) and [accepted your referral](accept-referral.md)**.  

Please note that you can refer multiple friends and get 200 Desmos Tokens per friend referred.

If one of the users who have referred does **not** create a post or accept your referral, you will **not** be granted any Desmos Token. 

->

This challenge requires you to refer one or more friends. A referral is deemed successful if below conditions have **all** been met:
- your referred friend has not joined Desmos Primer before your referral, and
- (s)he has completed the [Post challenge](create-post.md) after your referral

You will get **200 Desmos Tokens per successful referral**. There is no ceiling for the number of successful referral you can make but there is a cap of 100,000 Desmos Tokens to be rewarded for Phase 1 as a whole. Once this cap has been reached, Phase 1 will end immediately. So, please grab the chances to make as many referrals as possible!